src/components/ApiLink.astro

```diff
@@ -0,0 +1,18 @@
+---
+const { href, title } = Astro.props;
+---
+<a
+  href={href}
+  target="_self"
+  class="group inline-flex justify-center items-center text-interactive-secondary hover:text-interactive-secondary-hover"
+>
+  <span class="underline">
+    {title}
+  </span>
+  <span>&nbsp;</span>
```

Comment:
Question: Why do you need this additional span?

Reply:
@MeganSteenkamp It puts a space between the API badge and the link text. I've lifted and shifted the original existing `ProseA` component here - worth chatting about whether we should consider ditching the API badge in a follow up 👍